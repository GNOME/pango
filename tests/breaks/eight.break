200 24.10 99% alpha10beta 100-10 greek-roman
