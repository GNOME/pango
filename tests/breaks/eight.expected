Text:         ⁦2⁩ ⁦0⁩ ⁦0⁩  [ ]  ⁦2⁩ ⁦4⁩ ⁦.⁩ ⁦1⁩ ⁦0⁩  [ ]  ⁦9⁩ ⁦9⁩  ⁦%⁩ [ ]  ⁦a⁩ ⁦l⁩ ⁦p⁩ ⁦h⁩ ⁦a⁩ ⁦1⁩ ⁦0⁩ ⁦b⁩ ⁦e⁩ ⁦t⁩ ⁦a⁩  [ ]  ⁦1⁩ ⁦0⁩ ⁦0⁩  ⁦-⁩  ⁦1⁩ ⁦0⁩  [ ]  ⁦g⁩ ⁦r⁩ ⁦e⁩ ⁦e⁩ ⁦k⁩  ⁦-⁩  ⁦r⁩ ⁦o⁩ ⁦m⁩ ⁦a⁩ ⁦n⁩  [0x0a]  
Breaks:     c  c c c    lc c c c c c    lc c c  c   lc c c c c c c c c c c c    lc c c c  c  c c    lc c c c c c  lc c c c c c       Lc
Whitespace:        x               x            x                          x                   x                             w       w 
Sentences:  bs                                                                                                               e       b 
Words:      bs     be   bs   e s   be   bs   be b   bs                     be   bs     be bs   be   bs         be bs         be      b 
Graphemes:  b  b b b    b  b b b b b    b  b b  b   b  b b b b b b b b b b b    b  b b b  b  b b    b  b b b b b  b  b b b b b       b 
Hyphens:                                               i i i i i     i i i                             i i i i       i i i i           
